bundles/automation/pom.xml

```diff
@@ -44,7 +44,9 @@
           <groupId>com.mycila</groupId>
           <artifactId>license-maven-plugin</artifactId>
           <configuration>
-            <header>src/etc/header_prosyst.txt</header>
+            <excludes>
+              <exclude>**/*.*</exclude>
```

Comment:
This does not seem to work.
At least on my system `mvn license:format` still changes the bundles/automation stuff.